internal/cardinality/cardinality.go

```diff
@@ -19,13 +19,13 @@ package cardinality
 // Cardinalities
 const (
 	NoResult = 0x6e
-	One      = 0x6f
+	Single   = 0x6f
```

Comment:
This should be called `AtMostOne` to be consistent with the server enum.  `single` and `multi` are simplified cardinality indicators for the full range of possible cardinalities, where `single` is compatible with {`zero`, `at_most_one`, `one`}, and `multi` is compatible with all of them.

Reply:
changed